deploy.py

```diff
@@ -94,7 +94,7 @@
             app_ldscript="nrf52840_layout.ld",
             app_address=0x40000,
             storage_address=0xC0000,
-            storage_size=0x40000,
+            storage_size=0x14000,
```

Comment:
Yes we could erase for the whole `unallocated_size` (which for now is correct because it's not used by anything else) but it would be a waste of erase cycles since we don't use it (unless whatever erases those pages doesn't do it if they are already erased).

Ideally `deploy.py` should read those values from the board instead of hard-coding them.

Reply:
It might be easier to do it the other way around: define the values in `deploy.py` which then passes them to the boards through environment variables.
It would require to adapt the `build.rs` files to make sure we rebuild in case the values have changed since last compile time.

Reply:
Yes that would work too.

Reply:
I will look into that when I work on A/B boards. Happy with both solutions for the `storage_size`, saving cycles makes sense.